### ISSUE TITLE
Add tensorflow/theano setup to Travis config

### ISSUE THREAD

**Issue by julia326** [hammerlab/vaxrank#133]
Right now tests fail with an mhctools dependency issue:

```
Traceback (most recent call last):
  File "/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/nose/loader.py", line 418, in loadTestsFromName
    addr.filename, addr.module)
  File "/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/home/travis/build/hammerlab/vaxrank/test/test_shell_script.py", line 6, in <module>
    from vaxrank.cli import main as run_shell_script
  File "/home/travis/build/hammerlab/vaxrank/vaxrank/cli.py", line 25, in <module>
    from mhctools.cli import (
  File "/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/mhctools/__init__.py", line 10, in <module>
    from .mhcflurry import MHCflurry
  File "/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/mhctools/mhcflurry.py", line 22, in <module>
    from mhcflurry import Class1AffinityPredictor
  File "/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/mhcflurry/__init__.py", line 15, in <module>
    from .class1_affinity_prediction.class1_neural_network import (
  File "/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/mhcflurry/class1_affinity_prediction/__init__.py", line 3, in <module>
    from .class1_neural_network import Class1NeuralNetwork
  File "/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/mhcflurry/class1_affinity_prediction/class1_neural_network.py", line 8, in <module>
    import keras.models
  File "/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/keras/__init__.py", line 3, in <module>
    from . import activations
  File "/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/keras/activations.py", line 4, in <module>
    from . import backend as K
  File "/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/keras/backend/__init__.py", line 73, in <module>
    from .tensorflow_backend import *
  File "/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/keras/backend/tensorflow_backend.py", line 1, in <module>
    import tensorflow as tf
  File "/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/tensorflow/__init__.py", line 24, in <module>
    from tensorflow.python import *
  File "/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/tensorflow/python/__init__.py", line 51, in <module>
    from tensorflow.python import pywrap_tensorflow
  File "/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/tensorflow/python/pywrap_tensorflow.py", line 52, in <module>
    raise ImportError(msg)
ImportError: Traceback (most recent call last):
  File "/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/tensorflow/python/pywrap_tensorflow.py", line 41, in <module>
    from tensorflow.python.pywrap_tensorflow_internal import *
  File "/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/tensorflow/python/pywrap_tensorflow_internal.py", line 28, in <module>
    _pywrap_tensorflow_internal = swig_import_helper()
  File "/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/tensorflow/python/pywrap_tensorflow_internal.py", line 24, in swig_import_helper
    _mod = imp.load_module('_pywrap_tensorflow_internal', fp, pathname, description)
ImportError: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.17' not found (required by /home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/tensorflow/python/_pywrap_tensorflow_internal.so)
Failed to load the native TensorFlow runtime.
```

**Comment by julia326** [comment]
(Addressed in #134)

**Comment by julia326** [comment]
This has been fixed
